model/registry/gen-ai.yaml

```diff
@@ -47,6 +47,12 @@ groups:
         brief: The unique identifier for the completion.
         examples: ['chatcmpl-123']
         tag: llm-generic-response
+      - id: operation.name
+        stability: experimental
+        type: string
+        brief: The name of the LLM operation request being made.
+        examples: ['chat.completions', 'embeddings', 'speech.generations', 'audio.transcriptions', 'audio.translations', 'image.generations']
```

Comment:
```suggestion
        examples: ['chat', 'embeddings', 'speech.generations', 'audio.transcriptions', 'audio.translations', 'image.generations']
```

Reply:
True, ideally it should be completions and chat.completions

Reply:
I find `chat.completions` to be highly confusing with `completions`. I'd stick with `chat` and `completions`